client/my-sites/google-my-business/stats/index.js

```diff
@@ -66,9 +66,22 @@ class GoogleMyBusinessStats extends Component {
 		} );
 	};
 
-	renderViewsTooltipForDatanum = ( datanum, dataSeries ) => {
+	renderViewsTooltipForDatanum = ( datanum, interval ) => {
 		const { value: valueCount, date } = datanum;
-		if ( dataSeries.length > 20 ) {
+		if ( interval === 'quarter' ) {
+			return this.props.translate(
+				'%(value)d view on week %(week)s',
+				'%(value)d views on week %(week)s',
+				{
+					count: valueCount,
+					args: {
+						value: valueCount,
+						week: moment( date ).format( 'D MMMM YYYY' ),
+					},
+					comment: 'How many views per week for a Google My Business location with date.',
```

Comment:
while this comment is fine it's probably not needed. if anything it's not specific to Google My Business and we could generalize it to views on a given week. at that, we might as well rename `value` to `viewCount` and I think it will be super clear with no need for a comment.